src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -4402,10 +4402,12 @@ public long pfcount(final byte[]... keys) {
     return client.getIntegerReply();
   }
 
+  @Override
   public ScanResult<byte[]> scan(final byte[] cursor) {
     return scan(cursor, new ScanParams());
   }
 
+  @Override
   public ScanResult<byte[]> scan(final byte[] cursor, final ScanParams params) {
     checkIsInMultiOrPipeline();
```

Comment:
@sazzad16 shouldn't this method just call the new method with null on Type?

Reply:
@gkorland done